utils/xmdp/src/xmdp.c

```diff
@@ -160,7 +161,8 @@ int main() {
     uint32_t numipv4;
     if (sscanf(host_ip, "0x%x", &numipv4) == 1) {
       // find occurence
-      for (int i = 0; i < seen_len; i++)
+      int i;
```

Comment:
there is a `i` already declared in 45 line, so for line 164 it's a bit of redundancy